tests/queries/0_stateless/02421_simple_queries_for_opentelemetry.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# This function takes 2 arguments:
+# $1 - query
+# $2 - trace id
+function executeQuery()
+{
+  ${CLICKHOUSE_CLIENT} --database=${CLICKHOUSE_DATABASE} --opentelemetry_start_trace_probability=1 --query_id $1 -nq "
```

Comment:
Btw., nice trick with passing a query_id explicitly. Didn't know this is possible.

Reply:
It works fine. The test case will find related span logs based on the given query id.